### PR TITLE
 Added RSC with Graph using node Js

### PR DESCRIPTION
**Pull request by VaraPrasad-MSFT**

### CODE REVIEW

File: samples/graph-rsc/nodeJs/auth.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    var deferred = Q.defer();
+    
+    var requestParams = {
+      grant_type: 'client_credentials',
+      client_id: process.env.ClientId,  
+      client_secret: process.env.ClientSecret,  
+      scope: 'https://graph.microsoft.com/.default'
+    };
+  
+    var url = "https://login.microsoftonline.com/"+process.env.tenantId+"/oauth2/v2.0/token";
```
Comment (Wajeed-msft) [inline samples/graph-rsc/nodeJs/auth.js:18]: Could you please update this to fetch it from Tab Context?

File: samples/graph-rsc/nodeJs/teamsManifest/manifest.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.8/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.8",
+  "version": "1.0.0",
+  "id": "f0a1c0e8-92a5-4519-a67d-5ffe54293cbf",
```
Comment (Wajeed-msft) [inline samples/graph-rsc/nodeJs/teamsManifest/manifest.json:5]: Could you please remove actual values for id, configurationUrl & webApplicationInfo.Id and put placeholders?

File: samples/graph-rsc/nodeJs/views/RSCDemo.ejs [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>RSC Demo</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (Wajeed-msft) [inline samples/graph-rsc/nodeJs/views/RSCDemo.ejs:12]: Could you please check and remove unused script references?

File: samples/graph-rsc/nodeJs/views/configure.ejs [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>RSC demo node js</title>
+    <style>
+        body {
+            margin: 0px;
+            padding: 0px;
```
Comment (Wajeed-msft) [inline samples/graph-rsc/nodeJs/views/configure.ejs:11]: Could you please move styles to separate file?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — COMMENTED** [review]
Could you please add entry for this sample in root readme file?

File: samples/graph-rsc/nodeJs/auth.js (hunk 1 above)
Comment (VaraPrasad-MSFT) [reply samples/graph-rsc/nodeJs/auth.js]: fixed.

File: samples/graph-rsc/nodeJs/teamsManifest/manifest.json (hunk 2 above)
Comment (VaraPrasad-MSFT) [reply samples/graph-rsc/nodeJs/teamsManifest/manifest.json]: removed

File: samples/graph-rsc/nodeJs/views/RSCDemo.ejs (hunk 3 above)
Comment (VaraPrasad-MSFT) [reply samples/graph-rsc/nodeJs/views/RSCDemo.ejs]: removed unused references

File: samples/graph-rsc/nodeJs/views/configure.ejs (hunk 4 above)
Comment (VaraPrasad-MSFT) [reply samples/graph-rsc/nodeJs/views/configure.ejs]: moved to other folder and using

**Review by Wajeed-msft — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**